### PR TITLE
Add custom env

### PR DESCRIPTION
**Pull request by Asuforce**
ref: #64 

custom-env option can be freely set in the fluent.conf created by the user.

As a use case, if you are using multiple modules, you may want to set app-specific values.

```
# CUSTOM_ENV=<hoge-front | hoge-api>
<filter access_log>
  @type record_transformer
  enable_ruby
  auto_typecast true
  <record>
    app_name "#{ENV['CUSTOM_ENV']}"
  </record>
</filter>
```

### CODE REVIEW

**Comment by Asuforce** [comment]
@h3poteto Could you review plz :)

**Comment by h3poteto** [comment]
This pull request is good, but it would be better if I can set arbitrary key-value for environment variables.

File: README.md [hunk 1]
```diff
@@ -212,8 +213,8 @@ Please specify these annotations to your pods like [this](example/deployment.yam
 - <a name="tag-prefix">`fluentd-sidecar-injector.h3poteto.dev/tag-prefix`</a> is prefix of received log's tag. Default is `app`. It is used in [here](https://github.com/h3poteto/docker-fluentd-forward/blob/master/fluent.conf#L9).
 - <a name="time-key">`fluentd-sidecar-injector.h3poteto.dev/time-key`</a> is fluentd configuration in [here](https://github.com/h3poteto/docker-fluentd-forward/blob/master/fluent.conf#L6). Default is `time`.
 - <a name="time-format">`fluentd-sidecar-injector.h3poteto.dev/time-format`</a> is fluentd configuration in [here](https://github.com/h3poteto/docker-fluentd-forward/blob/master/fluent.conf#L7). Default is `%Y-%m-%dT%H:%M:%S%z`.
-- <a name="log-format">`fluentd-sidecar-injector.h3poteto.dev/log-format`</a> is fluentd configuration in [here](https://github.com/h3poteto/docker-fluentd-forward/blob/master/fluent.conf#L5). Default is `json`.
```
Comment (h3poteto) [inline README.md:215]: Why did you delete this line?

Comment (Asuforce) [reply README.md:215]: Sorry my mistake. I fixed it.

**Comment by Asuforce** [comment]
> This pull request is good, but it would be better if I can set arbitrary key-value for environment variables.

@h3poteto Yes, I think so too. I think the next step is to get to work on that.
